### ISSUE TITLE
Compatibility with Rails 4.2

### ISSUE THREAD

**Issue by davetoxa** [sumoheavy/jira-ruby#96]
```ruby
Bundler could not find compatible versions for gem "activesupport":
  In Gemfile:
    jira-ruby (>= 0) ruby depends on
      activesupport (~> 4.1.4) ruby
```

**Comment by jiaqingw** [comment]
any timeframe to resolve this?

**Comment by ranxiang** [comment]
Yes, please fix it.

**Comment by bobbrodie** [comment]
@davetoxa Which version of the gem are you using? I have version 0.1.10 working locally with Rails 4.2.0.

**Comment by andrewhavens** [comment]
@davetoxa 0.1.10 is the latest release that supports Rails 4.2.0. Newer releases of `jira-ruby` do not.

@bobbrodie The reason you can use `jira-ruby` 0.1.10 with Rails 4.2.0 is because the gemspec does not specify a specific `activesupport` version number (https://github.com/sumoheavy/jira-ruby/blob/v0.1.10/jira-ruby.gemspec#L23). 0.1.10 is actually a few versions behind the latest release (0.1.12) which locks down `activesupport` to the 4.1.x branch: (https://github.com/sumoheavy/jira-ruby/blob/v0.1.12/jira-ruby.gemspec#L23).

The solution for this is to either drop the `activesupport` version constraint, or if the version is important, make it more permissive (e.g. `>= 4.1.4`).

**Comment by bobbrodie** [comment]
You're so right! I forgot we pushed 0.1.11. I'm working on 0.1.13 now.

**Comment by bobbrodie** [comment]
Fixed in 0.1.13!

Thanks @andrewhavens
